src/common/types.rs

```diff
@@ -227,30 +227,30 @@ impl Ty
   }
 }
 
-pub fn u(size: u32) -> &'static Ty
+pub fn u(size: usize) -> &'static Ty
 {
   match size {
-    8  => U8,
+    1  => U8,
     _  => panic!("unsupported integer type: i{}", size)
   }
 }
 
-pub fn i(size: u32) -> &'static Ty
+pub fn i(size: usize) -> &'static Ty
```

Comment:
It was intended to be equivalent to llvm types which are globally used in this project. So, bit size is my intention.